anti-analysis/anti-debugging/debugger-detection/check-for-time-delay-via-gettickcount.yml

```diff
@@ -4,7 +4,9 @@ rule:
     namespace: anti-analysis/anti-debugging/debugger-detection
     authors:
       - michael.hunhoff@mandiant.com
-    scope: function
+    scopes:
+      static: function
+      dynamic: process
```

Comment:
this rule also cannot match in dynamic scope due to it relying on a basic block match